README.md

```diff
@@ -20,6 +20,7 @@ Features:
 * Optional module for public key recovery.
 * Optional module for ECDH key exchange.
 * Optional module for Schnorr signatures according to [BIP-340](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki).
+* Optional module for ElligatorSwift encoding for public keys according to [BIP-324](https://github.com/bitcoin/bips/blob/master/bip-0324.mediawiki).
```

Comment:
Thanks, this is missing!

Perhaps this is a bit simpler and more comprehensive:
```suggestion
* Optional module for ElligatorSwift key exchange according to [BIP-324](https://github.com/bitcoin/bips/blob/master/bip-0324.mediawiki).
```